js/components/closed-caption-multi-audio-menu/helpers.js

```diff
@@ -0,0 +1,150 @@
+var iso639 = require('iso-639-3');
+var CONSTANTS = require('../../constants/constants');
+var _ = require('underscore');
```

Comment:
Do we know how much this increases our file size? Underscore was only used in this project as a dev dependency until it was recently introduced with the VR360 updates. I recommend using the ES5 array methods directly (which are supported in all the browsers we support) and importing individual lodash functions as needed. We already bundle underscore in the core.min.js script, so using underscore in this project will add duplicate code

Reply:
That's a really good comment, thank you! 

Here's `ls -l` output on html5-skin

| filename | With underscore | Without underscore |
| --- | --- | --- |
| html5-skin.js | 2633829 | 2622434 |
| html5-skin.min.js | 1420095 | 1365399 |


It think it's a great idea to use ES5 array methods directly which are supported by the browsers, but the it breaks the tests, because the version of node that we're currently on (I believe it should be somewhere below `0.10.48`) doesn't support these methods in array prototype [link](https://node.green/#ES2015-subclassing-Array-is-subclassable-Array-prototype-filter). I think when we move to more recent version of node we could remove underscore as a dep.

For now I think it could be better to stick with not introducing more dependencies, since underscore is used in [viewControlsVr.js](https://github.com/ooyala/html5-skin/blob/a3ac20b2952bae4e938caf1577c9e3b91f3ea163/js/components/viewControlsVr.js#L5) anyway.